packages/waku/package.json

```diff
@@ -67,11 +67,12 @@
   ],
   "scripts": {
     "dev": "swc src -d dist -w --strip-leading-paths",
-    "test": "vitest run",
+    "test": "vitest run && pnpm run test:types",
```

Comment:
Hmm, can we simply add a reference in the root tsconfig.json instead?

Reply:
If we do that, when we run `compile:types` it would compile all of our `tests` and their types to the `dist` output. This seemed very undesirable, which is why I split them into separate configs, where I can set `noEmit: true` for the tests.

Reply:
Yeah, we don't want to emit test files for sure.
Let's see if @himself65 has an idea.

Reply:
Seems like there's no way to check file type without emitting a file

Related issue: https://github.com/microsoft/TypeScript/issues/40431

Reply:
thanks for checking out.

Reply:
there is a workaround, you can set output dir into a single directory, like `../../../dist`?

Reply:
I don't think I follow... can you explain the workaround a bit more?